scout/server/app.py

```diff
@@ -361,6 +361,22 @@ def list_remove_none(in_list: list) -> list:
         """Returns a list after removing any None values from it."""
         return [item for item in in_list if item is not None]
 
+    @app.template_filter()
+    def spliceai_max(values) -> Optional[float]:
+        """Returns a list of SpliceAI values, extracting floats only from values like 'score:0.23'."""
+        float_values = []
+        for value in values:
+            if value in [None, "-"]:
+                continue
+            if isinstance(value, str):
+                if "None" in value:
+                    continue
+                if ":" in value:  # Variant hits multiple genes
+                    value = value.split(":")[1].strip()
```

Comment:
We would also need to deal with "this_gene:-, that_gene:-", right?

Reply:
Right, I've pushed a fix now